File: Pattern1.cpp
//iNo = 6
//Pattern is:
//*       *       *       *       *       *

#include<iostream>
using namespace std;

class Pattern
{
    public:
    int iNo;

    Pattern(int iValue)
    {
        iNo = iValue;
    }

    void DisplayPattern()
    {
        int iCnt = 0;

        if(iNo < 0)
        {
            iNo =- iNo;
        }

        cout<<"Pattern is:\n";
        for(iCnt =1; iCnt <=iNo;iCnt++)
        {
            cout<<"*\t";
        }
        cout<<"\n";
    }
};

int main()
{
    int iNo =0;

    cout<<"Enter the number to print *:\n";
    cin>>iNo;

    Pattern pobj(iNo);

    pobj.DisplayPattern();

    return 0;
}

File: Pattern1.java
//iNo = 5
//Pattern is:
//*       *       *       *       *

import java.util.*;

class Pattern
{
    public int iNo;

    public Pattern(int iValue)
    {
        iNo = iValue;
    }

    public void DisplayPattern()
    {
        int iCnt = 0;

        if(iNo < 0)
        {
            iNo =- iNo;
        }

        System.out.println("Pattern is:");

        for(iCnt =1; iCnt <= iNo; iCnt++)
        {
            System.out.print("*\t");
        }
        System.out.println();

    }
}

class Pattern1
{
    public static void main(String arg[])
    {
        Scanner sobj = new Scanner(System.in);

        int iNo = 0;

        System.out.println("Enter the number to print * pattern:");
        iNo = sobj.nextInt();

        Pattern pobj = new Pattern(iNo);

        pobj.DisplayPattern();
    }
}